programs/ambassadors/ambassadors-schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "name": {
+        "type": "string"
+      },
+      "img": {
+        "type": "string",
+        "format": "uri"
+      },
+      "bio": {
+        "type": "string"
+      },
+      "title": {
+        "type": "string"
+      },
+      "github": {
+        "type": "string"
+      },
+      "twitter": {
+        "type": "string"
+      },
+      "linkedin": {
+        "type": "string"
+      },
+      "company": {
+        "type": "string"
+      },
+      "country": {
+        "type": "string"
+      },
+      "contributions": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "type": {
+              "type": "string",
+              "enum": ["article", "talk", "video", "other"]
```

Comment:
We have pending PRs which have more types than this.
@benjagm did you want to expand this list (and/or the list in the document)?

Reply:
Lets add:
- book
- paper
- initiative
- project
- working group